packages/contracts/markets/optimism/commons.ts

```diff
@@ -195,7 +195,7 @@ export const CommonsConfig: ICommonConfiguration = {
     [eOptimismNetwork.optimism]: '0x4200000000000000000000000000000000000006',
   },
   ReserveFactorTreasuryAddress: {
-    [eOptimismNetwork.optimism]: '0xEFd23344F89F7215417DC6F9E9627aBFfE69d2EE',
+    [eOptimismNetwork.optimism]: '0x6BfCF01CcdbA163c2A01F4DD99508790aBc20509',
```

Comment:
what even is the ReserveFactorTreasuryAddress

Reply:
I don't see this field being used anywhere in our repo

Reply:
I removed it, we use VMEXTreasury instead